temporalio/workflow.py

```diff
@@ -1051,6 +1063,34 @@ def get_current_details() -> str:
     return _Runtime.current().workflow_get_current_details()
 
 
+def has_last_completion_result() -> bool:
+    """Get the last completion result of the workflow. This be None if there was
+    no previous completion or the result was None
```

Comment:
```suggestion
    """Get the last completion result of the workflow. This will be None if there was
    no previous completion or if the result was None
```

Reply:
The return type should be `Optional[bool]` if it is to return `None`. That should  cause a type check error somewhere in tests I would think.

Reply:
This is probably just bad documentation copy pasta. This is just a `bool` and the docs need to be fixed.